content/300-guides/050-database/300-seed-database.mdx

```diff
@@ -5,103 +5,65 @@ metaDescription: Learn how to seed your database using Prisma's integrated seedi
 
 </TopBlock>
 
 ## Requirements for seeding with TypeScript or JavaScript
 
-Prisma's integrated seeding functionality expects the following:
-
-<!-- prettier-ignore-start -->
-
-1. A `seed.ts` or `seed.js` file in the [same location as your `schema.prisma` file](../../concepts/components/prisma-schema#prisma-schema-file-location).
-2. **One** of the following inside the `seed.js`/`seed.ts` file:
-
-    - A script inside `seed.ts` / `seed.js`
-    - An export named `seed` ([2.23.0](https://github.com/prisma/prisma/releases/tag/2.23.0) and later):
-
-      <TabbedContent tabs={[<FileWithIcon text="TypeScript" icon="code"/>, <FileWithIcon text="JavaScript" icon="code"/>]}>
-      <tab>
-
-      ```ts
-      export async function seed() {
-        // Prisma create query to seed models in database
-      }
-      ```
-
-      </tab>
-      <tab>
+Prisma's integrated seeding functionality expects a command in the `"seed"` key in the `"prisma"` key of your `package.json` file. This points to a file which holds your seed script. The location of the file is optional, although we recommend placing it in the `/prisma` folder by convention.
 
-      ```js
-      async function seed() {
-        // Prisma create query to seed models in database
-      }
+<TabbedContent tabs={[<FileWithIcon text="TypeScript" icon="file"/>, <FileWithIcon text="JavaScript" icon="file"/>]}> 
 
-      module.exports = {
-        seed
-      }
-      ```
+ 
+<tab> 
 
-      </tab>
-      </TabbedContent>
-
-    - A default export ([2.23.0](https://github.com/prisma/prisma/releases/tag/2.23.0) and later):
+ 
+```json
+"prisma": {
+  "seed": "ts-node prisma/seed.ts"
+},
+```
 
-      <TabbedContent tabs={[<FileWithIcon text="TypeScript" icon="code"/>, <FileWithIcon text="JavaScript" icon="code"/>]}>
-      <tab>
+ 
+</tab> 
 
-      ```ts
-      async function seed() {
-        // Prisma create query to seed models in database
-      }
-      export default seed;
-      ```
+ 
+<tab> 
 
-      </tab>
-      <tab>
+ 
+```json
+"prisma": {
+  "seed": "node prisma/seed.js"
+},
+```
 
-      ```js
-      async function seed() {
-        // Prisma create query to seed models in database
-      }
+ 
+</tab> 
 
-      module.exports = seed
-      ```
+ 
+</TabbedContent>
 
-      </tab>
-      </TabbedContent>
 
-<!-- prettier-ignore-end -->
 
-3. Optional [compiler options](#compiler-options) if you are using TypeScript.
+## Seeding your database with TypeScript or JavaScript
 
-## Seeding your database with TypeScript
+<TabbedContent tabs={[<FileWithIcon text="TypeScript" icon="file"/>, <FileWithIcon text="JavaScript" icon="file"/>]}> 
 
-To seed your database with TypeScript:
+ 
+<tab> 
 
+ 
 1. Create a new file named `seed.ts` in the same location as your `schema.prisma`.
```

Comment:
The location of this file is not define, the user can put it anywhere they want.